### PR TITLE
fix the contribx sig time

### PR DESCRIPTION
**Pull request by grodrigues3**
Fix the timezone/DST issue for contribx

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -315,8 +315,8 @@ sigs:
       company: CoreOS
     meetings:
     - day: Wednesday
-      time: "16:30"
-      tz: "UTC"
+      time: "9:30"
+      tz: "PST"
```
Comment (cblecker) [inline sigs.yaml]: This needs to be `"PT (Pacific Time)"` for the conversion link to work

**Comment by grodrigues3** [comment]
Fixed!  I noticed that yesterday and forgot to push.  Thanks for the careful review.

**Pull request merged**